.changeset/few-peas-hunt.md

```diff
@@ -0,0 +1,7 @@
+---
+'@astrojs/markdoc': minor
+'@astrojs/markdown-remark': minor
+'astro': minor
+---
+
+Update internal `shiki` syntax highlighter to `shikiji`. Existing docs on `shiki` still stays the same, but `shikiji` will help make bundling simpler and easier to maintain.
```

Comment:
Are there APIs that change? If not, maybe this should be a patch. If yes, then we should briefly document what's changed.

Reply:
The external APIs are unchanged, only the inner workings. I put `minor` since it feels like a feature, just that its not very obvious 🤔